modules/declarative-configuration-resource-create.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-role-based-access-control-3630.adoc
+:_content-type: PROCEDURE
+[id="declarative-configuration-resource-create_{context}"]
+= Creating declarative configurations
+
+Use `roxctl` to create the YAML files that store the configurations, create a config map from the files, and apply the config map.
+
+.Prerequisites
+* You have added the mount for the config map or secret during the installation of Central. In this example, the config map is called "declarative-configs". See the installation documentation listed in the "Additional resources" section for more information.
+
+.Procedure
+
+. Create the permission set by entering the following command. This example creates a permission set named "restricted" and is saved as the `permission-set.yaml` file. It sets `read` and `write` access for the `Administration` resource and `read` access to the `Access` resource.
++
+[source,terminal,subs=attributes+]
+----
+$ roxctl declarative-config create permission-set \
+--name="restricted" \
+--description="Restriction permission set that only allows \
+access to Administration and Access resources" \
+--resource-with-access=Administration=READ_WRITE_ACCESS \
+--resource-with-access=Access=READ_ACCESS > permission-set.yaml
+----
+. Create the role that allows access to the `Administration` and `Access` resources by entering the following command. This example creates a role named "restricted" and is saved as the `role.yaml` file.
++
+[source,terminal,subs=attributes+]
+----
+$ roxctl declarative-config create role \
+--name="restricted" \
+--description="Restricted role that only allows access to Administration and Access" \
+--permission-set="restricted" \
+--access-scope="Unrestricted" > role.yaml
+----
+. Create a config map from the two YAML files that were created in the earlier steps by entering the following command. This example creates the `declarative-configurations` config map.
++
+[source,terminal,subs=attributes+]
+----
+$ kubectl create configmap declarative-configurations \ <1>
+--from-file permission-set.yaml --from-file role.yaml \
+-o yaml --namespace=stackrox > declarative-configs.yaml
+----
+<1> For {ocp}, use `oc create`.
+. Apply the config map by entering the following command:
++
+[source,terminal,subs=attributes+]
+----
+$ kubectl apply -f declarative-configs.yaml <1>
+----
+<1> For {ocp}, use `oc apply`.
++
+After you apply the config map, Central retrieves configuration information from it and creates the resources. Central interacts with declarative configurations using these intervals:
```

Comment:
I think one thing to mention is that there might be a slight delay once the changes from the config map are propagated to central, so something along the lines of `After you apply the config map and the changes are propagated to the Central mount`? People might think that after applying the config map the resources will be there instantly (since the watch interval is 5 seconds) however we've seen that Kubernetes / Openshift takes a little bit of time until the actual changes are propagated to Central, hence there might be a slight delay.